### ISSUE TITLE
Consider alternative projection

### ISSUE THREAD

**Issue by wapratt** [wapratt/Assignment1#8]
Change to a Robinson or other preferred projection. I received an error message that x-ticks cannot use a non-rectangular projection. Perhaps I should remove ticks in favor of better projection?

**Comment by luisortizuriarte** [comment]
Yes, you would have to remove them. But that is a preference rather than a must-do.
